src/core/crypto/CryptoHelper.spec.ts

```diff
@@ -156,4 +158,39 @@ describe("CryptoHelper", () => {
             expect(cryptoHelper.sha256Base32Short(test.data)).toBe(test.output);
         }
     });
+    
+    it("should not call initialize if it has been initialized already", () => {
```

Comment:
The test title does not appear to reflect what is actually being tested. If I read the test code correctly what is being tested is if the `isInitialized` property is correctly being set not if the `initialize` method is not called twice.

Maybe change the test title or the test code?